docs/userguide/main.md

```diff
@@ -1167,4 +1167,47 @@ spec:
         serverName: "testserver.test.local"
         insecureSkipVerify: true
   # ...
+
+### Managing ISM policies with Kubernetes resources
+
+The operator provides a custom Kubernetes resource that allow you to create/update/manage ISM policies using Kubernetes objects.
+
+#### OpenSearch ISM policy
+
+It is possible to manage OpenSearch ISM policies in Kubernetes with the operator. The operator will not modify policies that already exist. You can create an example policy as follows:
+
+```yaml
+apiVersion: opensearch.opster.io/v1
+kind: ISMPolicy
+metadata:
+   name: sample-policy
+spec:
+   opensearchCluster:
+      name: my-first-cluster
+   description: Sample policy
+   policyId: sample-policy
```

Comment:
Add an extra comment about policyId being optional and metadata.name being used by default.